src/platform/loader-base.ts

```diff
@@ -82,6 +83,13 @@ export abstract class LoaderBase {
     }
     return this.loadFile(path);
   }
+  /** https://regex101.com/r/0qpxfW/2 */
+  isJvmClasspath(candidate: string): boolean {
+    return /^(?:[a-z]\w*|[a-z]\w*\.)+(?:[A-Z]\w*|[A-Z]\w*\.)+$/.test(candidate);
```

Comment:
Question for the reviewer (@piotrswigon): This regex is correct and tested, but somewhat slow. This causes a few TS tests to fail with timeouts in Travis. 

Should I re-write this logic with string manipulation?

Reply:
An additional benefit is that explicit code will be far far easier to understand. I say go for it!

Reply:
It seems a little suspicious that the regex could be slow enough to cause timeouts.
My guts says that the regex isn't the issue and its a flake.

(Regex compilation is fascinating btw, they tend to produce incredibly fast code
More info for the interested: https://blog.chromium.org/2009/02/irregexp-google-chromes-new-regexp.html )

Reply:
Thanks for the links! I've been burnt by [unexpectedly slow](https://stackoverflow.com/questions/14205096/c11-regex-slower-than-python) regex implementations in the past. 

I'll probably just proceed with what's here, thanks for the review.